File: LLM-Translations/GPT-4/codenet/Java/C/s902370101.c
#include <stdio.h>

int main() {
    int n, max = 0, min = 10000;
    scanf("%d", &n);
    int array[n];

    for (int i = 0; i < n; i++) {
        scanf("%d", &array[i]);
        if (max < array[i]) {
            max = array[i];
        }

        if (min > array[i]) {
            min = array[i];
        }
    }
    printf("%d\n", max - min);

    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Java/Go/s902370101.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	fmt.Scan(&n)
	array := make([]int, n)
	max := math.MinInt32
	min := math.MaxInt32

	for i := 0; i < len(array); i++ {
		fmt.Scan(&array[i])
		if max < array[i] {
			max = array[i]
		}
		if min > array[i] {
			min = array[i]
		}
	}
	fmt.Println(max - min)
}

// 
